src/main/java/core/basesyntax/GeometricFigureSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class GeometricFigureSupplier {
+    public GeometricFigure getRandomFigure() {
+        int index = new Random().nextInt(0, GeometricFigure.values().length);
```

Comment:
Creating a new `Random` object each time can lead to a less uniform distribution of random numbers if the method is called in rapid succession. It's better to create a single `Random` instance for the class.